index.html

```diff
@@ -14,41 +14,88 @@
     <link href="https://fonts.googleapis.com/css2?family=Roboto+Mono&display=swap" rel="stylesheet">
     <link rel="icon" href="res/favicon.ico" type="image/x-icon" />
     <link rel="shortcut icon" href="res/favicon.ico" type="image/x-icon" />
+    <link href="https://fonts.googleapis.com/css2?family=Material+Icons+Outlined" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/bdbf02c54e.js" crossorigin="anonymous"></script>
 </head>
 
 <body>
+
+    <div id="preloader"></div>
+
     <header></header>
 
     <main>
         <div id="about" class="nav-div" data-label="About">
-            <!-- <div id="img-div"> -->
-            <img id="logo" src="res/logo.png"></img>
-            <!-- </div> -->
-            <p id="about-para" class="nav-para">about</p>
+            <div class="section-content">
+                <div id="about-container">
+                    <img id="logo" src="res/logo.png"></img>
+                    <p id="about-content">
+                        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vivamus semper sem eu volutpat dapibus. Etiam lorem orci, efficitur et tellus sit amet, placerat fringilla orci.
+                    </p>
+                </div>
+                <p id="about-para" class="nav-para">about</p>
+            </div>
         </div>
+
+
         <div id="projects" class="nav-div" data-label="Projects">
-            <p id="projects-para" class="nav-para">projects</p>
-        </div>
-        <div id="contact" class="nav-div" data-label="Contact">
-            <p id="contact-para" class="nav-para">contact</p>
-        </div>
-        <nav class="nav">
-            <div class="nav-item">
-                <a href="#about" class="nav-link">Bottle</a>
-                <span class="nav-label">About</span>
+            <div class="section-content">
+                <div id="projects-container">
+                    <img id="wolf-logo" src="res/wolf.png">
+                    <p>
+                        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vivamus semper sem eu volutpat dapibus. Etiam lorem orci, efficitur et tellus sit amet, placerat fringilla orci. Donec facilisis lectus nisl, eget tincidunt lorem ornare ut. Sed ac lorem nulla.
+                        Sed tincidunt interdum augue, sed gravida nunc aliquam at. Suspendisse dictum cursus neque, vel finibus nunc. Ut sed lectus vitae quam porttitor commodo in vel nibh. Nam faucibus ut erat blandit molestie. Donec egestas sed nunc
+                        ac accumsan.
+                    </p>
+                </div>
+                <!-- <i id="projects-icon" class="far fa-edit"></i> -->
```

Comment:
What's this comment for? Please check all of your code and remove redundant comments.